src/main/java/ch/njol/skript/expressions/ExprClicked.java

```diff
@@ -58,17 +58,17 @@
 import ch.njol.util.coll.CollectionUtils;
 
 @Name("Clicked Block/Entity/Inventory/Slot")
-@Description("The clicked block, entity, inventory slot, inventory, type or action.")
+@Description("The clicked block, entity, inventory slot, inventory and the inventory type or action.")
```

Comment:
Did you mean "inventory click type"?

Reply:
```autohotkey
[the] ( clicked (block|%*itemtype/entitydata%)|clicked slot|clicked inventory|click (type|action)|inventory action) 
```
Looks like `click action` didn't get removed

```suggestion
@Description("The clicked block, entity, inventory, inventory slot, inventory click type or inventory action.")
```